code/modules/research/xenoarchaeology/artifact/artifact_unknown.dm

```diff
@@ -323,3 +324,12 @@
 		my_effect.UpdateMove()
 	if(secondary_effect)
 		secondary_effect.UpdateMove()
+// BEGIN_INTERNALS
+// END_INTERNALS
+// BEGIN_FILE_DIR
+#define FILE_DIR .
+// END_FILE_DIR
+// BEGIN_PREFERENCES
+// END_PREFERENCES
+// BEGIN_INCLUDE
+// END_INCLUDE

```

Comment:
This isn't supposed to be here.

Reply:
That has always been in the code, no idea why it's listed as a change. I only touched effects/